### PR TITLE
Added required features for the clone

### PR DESCRIPTION
**Pull request by TedLivist**
## The features included:
- aside tag for editors' picks
- navbar
- footer
- images
- page styling

## The following files are also included:
- index.html file - this contains the html elements
- style.css file - for styling the html file
- README file - for highlighting some description about the project

### CODE REVIEW

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @TedLivist      ! :wave: :wave:

#### You have done well so far, Your project looks really good!  :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.
- - -

### **Recommended**

**General**
- [ ] Please add a live link to your readme file.

**Section Specific requirements**
- [ ] Your navbar is missing the bottom borderline. Please add it.
 
- [ ] Some parts of the main content texts have fonts that are larger than what is on the page. I'll recommend using [this](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej) chrome extension tool to help with this.

- [ ] The theory of inflation's beginning paragraph should be black.
- [ ] The Editor's Picks section should be beside the paragraph with a short thank you note on it.

**Original**

Screenshot (123)

- [ ] The copyright section below the footer should be black font. Certain parts of your page should be grey/black instead of orange.


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,646 @@
...
+  <link rel="stylesheet" href="index.css">
+  <title>Space Ripples Reveal Big Bang's Smoking Gun - New York Times</title>
+  <link rel="shortcut icon" type="image/jpg" href="images/digital.jpg">
+</head>
+<body>
+  <nav>
+    <i class="fas fa-bars"></i>
+    <i class="fas fa-search"></i>
+    <span id="space-text">SPACE & COSMOS</span>
+    <img src="images/nyheader.jpg" alt="nyt-header">
```
Comment (teekaytech) [inline index.html]: The NY Times logo should be linked. i.e. use `<a>` tag to wrap the logo  (<a href="https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0" rel="nofollow noreferrer noopener" target="_blank">link to page</a>)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+            <a href="#"><div class="subscribe-text">Email Newsletters</div></a>
+            <a href="#"><div class="subscribe-text">Corporate Subscriptions</div></a>
+            <a href="#"><div class="subscribe-text">Education Rate</div></a>
+            <hr>
+            <a href="#"><div class="subscribe-text">Mobile Applications</div></a>
+            <a href="#"><div class="subscribe-text">Replica Edition</div></a>
+            <a href="#"><div class="subscribe-text">International</div></a>
+            <a href="#"><div class="subscribe-text">Canada</div></a>
+            <a href="#"><div class="subscribe-text">Espanol</div></a>
+            <a href="#"><div class="subscribe-text"><img src="images/chinese.jpg" alt=""></div></a>
```
Comment (teekaytech) [inline index.html]: Always give the `alt` attribute a descriptive value.

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

### Congratulations @TedLivist :tada: :clinking_glasses: 

_**Thank you for implementing the required changes by the previous reviewer**_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

**Comment(s)**
- Kindly check the inline comments below and commit the changes before you merge.

Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html (hunk 1 above)
Comment (TedLivist) [reply index.html]: Thanks for the review. I have corrected this

File: index.html (hunk 2 above)
Comment (TedLivist) [reply index.html]: I have corrected this also

**Comment by TedLivist** [comment]
> # Status: Approved 🟢
> ### Congratulations @TedLivist 🎉 🥂
> _**Thank you for implementing the required changes by the previous reviewer**_
> 
> **This project has passed all the mandatory requirements 👏 It is therefore approved ✔️**
> 
> **Comment(s)**
> 
> * Kindly check the inline comments below and commit the changes before you merge.
> 
> Again, congratulations, you can merge to the default branch.
> Do not forget to fill the project completion form, as this is important for your weekly progress score.
> 
> **_Good luck with your next task._**
> 
> **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> **Happy Coding 😃**

Hello Taofeek, I am kinda confused with the project instructions.
The Pathwright Project instruction says we should not merge.

dont-merge

Yet according to your review, I can now merge. I do not know which one of the guidelines to follow.

**Comment by teekaytech** [comment]
> > # Status: Approved green_circle
> > ### Congratulations @TedLivist tada clinking_glasses
> > _**Thank you for implementing the required changes by the previous reviewer**_
> > **This project has passed all the mandatory requirements clap It is therefore approved heavy_check_mark**
> > **Comment(s)**
> > 
> > * Kindly check the inline comments below and commit the changes before you merge.
> > 
> > Again, congratulations, you can merge to the default branch.
> > Do not forget to fill the project completion form, as this is important for your weekly progress score.
> > **_Good luck with your next task._**
> > **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> > **Happy Coding smiley**
> 
> Hello Taofeek, I am kinda confused with the project instructions.
> The Pathwright Project instruction says we should not merge.
> 
> dont-merge
> 
> Yet according to your review, I can now merge. I do not know which one of the guidelines to follow.

Hi @TedLivist

Kindly give me some time to make some clarifications about the instruction. I'll get back to you soon.

**Comment by teekaytech** [comment]
> > > # Status: Approved green_circle
> > > ### Congratulations @TedLivist tada clinking_glasses
> > > _**Thank you for implementing the required changes by the previous reviewer**_
> > > **This project has passed all the mandatory requirements clap It is therefore approved heavy_check_mark**
> > > **Comment(s)**
> > > 
> > > * Kindly check the inline comments below and commit the changes before you merge.
> > > 
> > > Again, congratulations, you can merge to the default branch.
> > > Do not forget to fill the project completion form, as this is important for your weekly progress score.
> > > **_Good luck with your next task._**
> > > **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> > > **Happy Coding smiley**
> > 
> > 
> > Hello Taofeek, I am kinda confused with the project instructions.
> > The Pathwright Project instruction says we should not merge.
> > dont-merge
> > Yet according to your review, I can now merge. I do not know which one of the guidelines to follow.
> 
> Hi @TedLivist
> 
> Kindly give me some time to make some clarifications about the instruction. I'll get back to you soon.

Hello @TedLivist 

I think there's a misunderstanding. The purpose of the message was to ask you not to merge before the PR was approved. Now that the PR is approved, I believe you can merge. I've reported the issue to the relevant channel and if it has not been fixed by now, the statement will be fixed soon.

Happy coding!

**Comment by TedLivist** [comment]
> > > > # Status: Approved green_circle
> > > > ### Congratulations @TedLivist tada clinking_glasses
> > > > _**Thank you for implementing the required changes by the previous reviewer**_
> > > > **This project has passed all the mandatory requirements clap It is therefore approved heavy_check_mark**
> > > > **Comment(s)**
> > > > 
> > > > * Kindly check the inline comments below and commit the changes before you merge.
> > > > 
> > > > Again, congratulations, you can merge to the default branch.
> > > > Do not forget to fill the project completion form, as this is important for your weekly progress score.
> > > > **_Good luck with your next task._**
> > > > **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> > > > **Happy Coding smiley**
> > > 
> > > 
> > > Hello Taofeek, I am kinda confused with the project instructions.
> > > The Pathwright Project instruction says we should not merge.
> > > dont-merge
> > > Yet according to your review, I can now merge. I do not know which one of the guidelines to follow.
> > 
> > 
> > Hi @TedLivist
> > Kindly give me some time to make some clarifications about the instruction. I'll get back to you soon.
> 
> Hello @TedLivist
> 
> I think there's a misunderstanding. The purpose of the message was to ask you not to merge before the PR was approved. Now that the PR is approved, I believe you can merge. I've reported the issue to the relevant channel and if it has not been fixed by now, the statement will be fixed soon.
> 
> Happy coding!

Okay then. I will merge. Thanks for the feedback.

**Pull request merged**